schema/appmodel/trigger.schema.xml

```diff
@@ -169,6 +170,7 @@
   <attribute name="td_out_of_timeout" type="bool" init-value="true" is-not-null="yes"/>
   <attribute name="buffer_timeout" type="u32" init-value="100" is-not-null="yes"/>
   <attribute name="td_readout_limit" type="u32" init-value="1000" is-not-null="yes"/>
+  <relationship name="latency_monitoring_conf" class-type="LatencyMonitoringConf" low-cc="zero" high-cc="zero" is-composite="yes" is-exclusive="yes" is-dependent="yes"/>
```

Comment:
Shouldn't this be
```suggestion
  <relationship name="latency_monitoring_conf" class-type="LatencyMonitoringConf" low-cc="one" high-cc="one" is-composite="no" is-exclusive="no" is-dependent="no"/>
```
`one`, otherwise we're saying "I want exactly no LatencyMonitoringConf objects", so what's the point of having them?
`is-composite="no"` because this is definitely not a composite class. `LatencyMonitoringConfig` is made of one boolean, not entirely of other objects.
`is-exclusive="no"` presumably because we want to share this object? Otherwise there's no point of making it a separate class in the first place.
`is-dependent="yes"` is one that I'm unsure about, I always assumed it's only for objects that go into composite classes. Perhaps @gcrone knows better here.